app/pnpm-lock.yaml

```diff
@@ -69,7 +75,7 @@ importers:
         version: 0.4.4
       sbc-common-components:
         specifier: 3.0.15-c
-        version: 3.0.15-c(patch_hash=ic3kkhec77kqvmbcac22ctyitm)(@types/node@20.19.18)(postcss@8.5.6)(sass@1.32.13)
+        version: 3.0.15-c(patch_hash=72b42f4950eb3521620faa668e06fd2cc7cb02af011625f2ecba78c4d6c3c01a)(@types/node@20.19.18)(postcss@8.5.6)(sass@1.32.13)
```

Comment:
I may need to re-create the sbc-common-components patch. I'm looking into it presently.

Reply:
I didn't re-create anything and it's working fine.

(I did delete my node_modules content and re-ran `pnpm i`.)

Reply:
It works locally but not in CI/CD scripts:

<img width="1151" height="469" alt="image" src="https://github.com/user-attachments/assets/60a57d4d-5d50-4dd9-aced-25b8a7a80794" />

The suggest pnpm command does not change any files locally (so I have nothing new to commit and thus it won't fix anything). I will try to re-create the patch after all...